drivers/hyperv/hyperv.go

```diff
@@ -35,6 +43,7 @@ const (
 	defaultVLanID               = 0
 	defaultDisableDynamicMemory = false
 	defaultSwitchID             = "c08cb7b8-9b3c-408e-8e30-5e16a3aeb444"
+	defaultTimeout              = time.Minute * 10
```

Comment:
awfully long time out...

Reply:
How long do you think it's appropriate? 3 or 5 min?

Reply:
a default of 3 is long enough, since when more is needed, the caller should override this.

Reply:
Sure.
Let me make the timeout configurable if needed.